Tasks/GooglePlayReleaseV4/modules/inputsHelper.ts

```diff
@@ -0,0 +1,179 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+
+import * as googleutil from './googleutil';
+import * as fileHelper from './fileHelper';
+
+/**
+ * Fills client key with valid parameters depending on auth type. Returns the filled key.
+ * @returns client key
+ */
+export function getClientKey(): googleutil.ClientKey {
+    const authType: string = tl.getInput('authType', true);
+    let key: googleutil.ClientKey = {};
+    if (authType === 'JsonFile') {
+        const serviceAccountKeyFile: string = tl.getPathInput('serviceAccountKey', true, true);
+
+        const stats: tl.FsStats = tl.stats(serviceAccountKeyFile);
+        if (stats && stats.isFile()) {
+            key = require(serviceAccountKeyFile);
+        } else {
+            tl.debug(`The service account file path ${serviceAccountKeyFile} points to a directory.`);
+            throw new Error(tl.loc('InvalidAuthFile', serviceAccountKeyFile));
+        }
+    } else if (authType === 'ServiceEndpoint') {
+        let serviceEndpoint: tl.EndpointAuthorization = tl.getEndpointAuthorization(tl.getInput('serviceEndpoint', true), false);
+        key.client_email = serviceEndpoint.parameters['username'];
+        key.private_key = serviceEndpoint.parameters['password'].replace(/\\n/g, '\n');
+    }
+
+    return key;
+}
+
+const actions = ['OnlyStoreListing', 'SingleBundle', 'SingleApk', 'MultiApkAab'] as const;
+export type Action = (typeof actions)[number];
+
+function isOfTypeAction(userInput: string): userInput is Action {
+  return (actions as readonly string[]).includes(userInput);
+}
```

Comment:
Could you please add some comments regarding checking of action type?

Reply:
https://github.com/microsoft/google-play-vsts-extension/pull/292/commits/e5f75f23ba4f707d16d4917cbd10fa4077978579